p4-16/psa/PSA.mdk

```diff
@@ -512,22 +511,15 @@ translation in the agent software, between target-specific values, and
 a value with a common unit and 0 value reference across all targts.
 ~
 
-~ TBD
-It is not clear yet whether it is reasonable to use a value of type
-`PortId_t` as an index to of a `Register` extern, or of an indexed
-`Counter` or `Meter` extern.  A likely result of attempting to do this
-in a P4 program is that the indexes accessed will be the data plane
-specific values of `PortId_t`, and when the control plane attempts to
-access values in these externs, the numerical translation may not be
-performed on the index.  It is reasonable to use a `DirectCounter` or
-`DirectMeter` with a table key that includes a `PortId_t` field,
-because the agent software will perform port translation for table
-keys.
-
-If this is not a reasonable thing to do, the example program
-`psa-example-counters.p4` should be modified so that it does not do
-this, and scrub all other examples for any similar behavior.
-~
```

Comment:
Removing this TBD could be done in a separate PR if you wish.  Antonin addressed this issue in P4Runtime v1.0 in Jan 2019.